sdk/provisioning/Generator/src/Generator.csproj

```diff
@@ -9,6 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="CommandLineParser" VersionOverride="2.9.1"/>
```

Comment:
VersionOverrides are intended only to work around temporary compatibility issues and should always be accompanied by an issue tracking their removal.  

Please DO NOT do overrides that are for any other reason.   We should first look to bump central versions and if that is not possible, we do a scoped rule in the central versions with a comment indicating why.


//cc: @ArthurMa1978, @ArcturusZhang